2_process.R

```diff
@@ -0,0 +1,48 @@
+source("2_process/src/pair_nhd_reaches.R")
+source("2_process/src/pair_nhd_catchments.R")
+source("2_process/src/create_GFv1_NHDv2_xwalk.R")
+source("2_process/src/write_data.R")
+source("2_process/src/write_ind_files.R")
+
+p2_targets_list <- list(
+  
+  # Pair PRMS segments (NHGFv1) with intersecting NHDPlusV2 reaches and contributing NHDPlusV2 catchments
+  tar_target(
+    p2_prms_nhdv2_xwalk,
+    create_GFv1_NHDv2_xwalk(prms_lines = p1_GFv1_reaches_sf,
+                            nhd_lines = p1_nhdv2reaches_sf,
+                            prms_hrus = p1_GFv1_catchments_sf,
+                            min_area_overlap = 0.5,
+                            drb_segs_spatial = drb_segs_spatial)
+  ),
+  
+  # Save GFv1-NHDv2 xwalk table 
+  tar_target(
```

Comment:
Could you save a csv or txt target, too? I could see someone wanting to check a single reach quickly by looking at the table

Reply:
I ended up saving the xwalk table as a csv instead of rds. This has the advantages of being human readable and is more language agnostic.